src/aims_ui/models/get_fields.py

```diff
@@ -164,13 +164,6 @@ def get_fields(endpoint_name):
         common_fields['england_boost'],
         common_fields['wales_boost'],
         common_fields['scotland_boost'],
-        Field(
-            'favourwelsh',
-            search_type='checkbox',
-            display_title='Flavour Welsh',
-            description=
-            'If you prefer the results to be flavoured Welsh tick this box',
```

Comment:
The favourwelsh flag is a tie-breaker for when the English and Welsh versions of the address have the same number of matching parts

Reply:
I have adjusted the "boost-checkbox-description" which is the one that's now shown!
 'Boost a region. This is used as a "tie breaker" when the same address from different regions is present.'